### PR TITLE
Fix relative path in feg docker compose file.

### PR DESCRIPTION
**Pull request by reddydodda**
1. this will make to deploy feg from any path using instalation script without magma repo.

2. copy configs from repo

Signed-off-by: reddydodda <reddydodda@gmail.com>

### CODE REVIEW

File: orc8r/tools/docker/install_gateway.sh [hunk 1]
```diff
@@ -92,6 +92,7 @@ mkdir -p /var/opt/magma/docker
 # Copy certs and configs
 cp rootCA.pem /var/opt/magma/certs/
 cp control_proxy.yml /etc/magma/
+cp -r "$INSTALL_DIR"/magma/feg/gateway/configs /var/opt/magma/
```
Comment (mpgermano) [inline orc8r/tools/docker/install_gateway.sh]: We shouldn't need this. By default, configs are included in the docker images at /etc/magma. /var/opt/magma/configs is only for config overrides.

File: feg/gateway/docker/docker-compose.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ services:
     <<: *goservice
     container_name: csfb
     build:
-      context: ../../../
+      context: ${MAGMA_GITHUB_URL}#${MAGMA_GITHUB_BRANCH}
```
Comment (mpgermano) [inline feg/gateway/docker/docker-compose.yml]:
```suggestion
      context: ${BUILD_CONTEXT}
```

File: feg/gateway/docker/docker-compose.yml [hunk 3]
```diff
@@ -112,3 +112,7 @@ services:
       sh -c "/usr/local/bin/generate_service_config.py --service=redis --template=redis &&
              /usr/bin/redis-server /var/opt/magma/tmp/redis.conf --daemonize no &&
              /usr/bin/redis-cli shutdown"
+
+volumes:
```
Comment (mpgermano) [inline feg/gateway/docker/docker-compose.yml]: We can remove these volumes as they're only intended for development environment. In the new prod_env file, you can set:
```
CERTS_VOLUME=/var/opt/magma/certs
CONFIGS_VOLUME=/var/opt/magma/configs
```

File: feg/gateway/docker/.env [hunk 4]
```diff
@@ -8,6 +8,8 @@
 COMPOSE_PROJECT_NAME=feg
 DOCKER_REGISTRY=feg_
 IMAGE_VERSION=latest
+MAGMA_GITHUB_URL=https://github.com/facebookincubator/magma.git
```
Comment (mpgermano) [inline feg/gateway/docker/.env]: We use this .env file for local development, so we shouldn't add the changes you've made here. Instead, can you create a new environment file called prod_env which has the changes you have here, and then in the install script, use prod_env. We can then use an environment variable to set the context to either ../../../ or https://github.com/facebookincubator/magma.git#master

I.e in .env:

```suggestion
BUILD_CONTEXT=../../../
```

And in prod_env
```suggestion
BUILD_CONTEXT=https://github.com/facebookincubator/magma.git#master
```

File: feg/gateway/docker/docker-compose.yml [hunk 5]
```diff
@@ -94,7 +94,7 @@ services:
     <<: *pyservice
     container_name: magmad
     build:
-      context: ../../../
+      context: ${MAGMA_GITHUB_URL}#${MAGMA_GITHUB_BRANCH}
```
Comment (mpgermano) [inline feg/gateway/docker/docker-compose.yml]:
```suggestion
      context: ${BUILD_CONTEXT}
```

**Review by mpgermano — CHANGES_REQUESTED** [review]
Thanks for the additions @reddydodda. Please see the changes requested.

File: feg/gateway/docker/docker-compose.yml (hunk 2 above)
Comment (reddydodda) [reply feg/gateway/docker/docker-compose.yml]: fixed

File: feg/gateway/docker/docker-compose.yml (hunk 3 above)
Comment (reddydodda) [reply feg/gateway/docker/docker-compose.yml]: fixed

File: feg/gateway/docker/docker-compose.yml (hunk 5 above)
Comment (reddydodda) [reply feg/gateway/docker/docker-compose.yml]: fixed

**Review by mpgermano — APPROVED** [review]
Looks good!

**Pull request closed without merging**